src/ms-store-badge.ts

```diff
@@ -130,12 +130,12 @@
         transition: 0.35s ease;
       }
             
-      img:hover {
-        transform: translate(0, -4px);
-        cursor: pointer;
-        box-shadow: 0 12px 40px 2px rgba(0, 0, 0, 0.05);
-        transition: 0.35s ease;
-      }
+      XXXXimg:hover {
```

Comment:
Can you just remove this altogether (put it in a separate branch if you need to work on it)? Line 138 might be invalid CSS (?) so I'm concerned it could break other CSS selectors under it.